test/cql-pytest/test_schema.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2024-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+
+# Tests for schema related things
```

Comment:
"schema related things" is very general, can contain tests for tables, views, functions, keyspaces and who knows what or why, so I don't think this makes a great test file. I would probably have put this in test_alter_table.py. But I we can always move this test later if you prefer to keep it this way.

Reply:
renamed the file